### PR TITLE
docs/readme-test-templates

### PR DESCRIPTION
**Pull request by bre1470**
Added detailes for test templates.

### CODE REVIEW

File: samples/readme.md [hunk 1]
```diff
@@ -40,7 +45,11 @@ The tests that are added to `/samples/unit-tests` are also part of the
 pre-commit tests that run via karma. Run `gulp test` on the root to pre-check.
 Read more at [highcharts/test](https://github.com/highcharts/highcharts/tree/master/test).
 
-### Useful tips for setting up tests
+
+
+Useful Tips for setting up Tests
```
Comment (TorsteinHonsi) [inline samples/readme.md]: Use either all caps or no caps, but consistently

File: samples/readme.md [hunk 2]
```diff
@@ -64,3 +77,80 @@ export the chart first.
...
+});
+```
+
+### Debugging a Test Template
+
+If you like to debug a template-based test in your browser with the help of
+`highcharts-utils`, make sure that there is a copy of the template in
+[`/public/javascript/test-templates`](https://github.com/highcharts/highcharts-utils/tree/master/public/javascript/test-templates).
+
+### Create a new Test Template
```
Comment (TorsteinHonsi) [inline samples/readme.md]: As above

File: samples/readme.md (hunk 1 above)
Comment (TorsteinHonsi) [reply samples/readme.md]: https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html

**Comment by bre1470** [comment]
Please review

**Pull request merged**